### PR TITLE
chore: added Troubleshooting section in AWS PowerShell Lambda Readme for enabling Debug output.

### PR DESCRIPTION
**Pull request by ashishdhingra**
*Issue #, if available:* DOTNET-7763

*Description of changes:*
Added Troubleshooting section in AWS PowerShell Lambda Readme for enabling Debug output.
___
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: PowerShell/README.md [hunk 1]
```diff
@@ -41,6 +41,25 @@ New-AWSPowerShellLambda|Used to create an initial PowerShell script that is base
...
+
+Kindly note that the default value of [DebugPreference](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_preference_variables?view=powershell-7.4#debugpreference) variable is `SilentlyContinue`, which means the debug message isn't displayed and execution continues without interruption. The `-Debug` parameter could be used to override the `$DebugPreference` value.
+
+Enabling output of `Write-Debug` to CloudWatch logs is a 2 step process:
+- In PowerShell Lambda script, 
+  - Either need to set `$DebugPreference = "Continue"` at the beginning of script. Thereafter use `Write-Debug` to output debug messages ; **OR**
+  - Include `-Debug` parameter while executing `Write-Debug` (e.g.` Write-Debug "Testing Lambda PowerShell Write-Debug" -Debug`).
+- At Lambda function level, set the value of `AWS_LAMBDA_HANDLER_LOG_LEVEL` environment variable with value `DEBUG`. This would enable debug logs at Lambda level. This environment variable could be set:
```
Comment (normj) [inline PowerShell/README.md:54]: We should tell users to set the log level in the AWS console or .NET Lambda Tool. We recently shipped new version of the tooling that allows setting the log level. https://github.com/aws/aws-extensions-for-dotnet-cli/blob/master/RELEASE.CHANGELOG.md#release-2024-10-09

File: PowerShell/README.md [hunk 2]
```diff
@@ -41,6 +41,25 @@ New-AWSPowerShellLambda|Used to create an initial PowerShell script that is base
...
+  - Either manually in Lambda function configuration in AWS console; **OR**
+  - While executing CmdLet `Publish-AWSPowerShellLambda`, passing parameter `-EnvironmentVariable @{'AWS_LAMBDA_HANDLER_LOG_LEVEL'='DEBUG'}`.
+  
+  The value of the `AWS_LAMBDA_HANDLER_LOG_LEVEL` environment variable is set to the values of the [LogLevel](https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/src/Amazon.Lambda.Core/ILambdaLogger.cs#L7) enum.
+
+Kindly note that the role assigned to Lambda function should have permissions to write to CloudWatch logs.
```
Comment (normj) [inline PowerShell/README.md:60]: You love your "Kindly note" prefixes 😄. You can be direct and say "The role assigned to Lambda ...".

**Review by normj — CHANGES_REQUESTED** [review]

File: PowerShell/README.md (hunk 1 above)
Comment (normj) [reply PowerShell/README.md:54]: Actually ignore my previous message. The Lambda API for setting log level is only when the format is set for `JSON`. .NET is special in that it supports log level for text format as well which has existed long before Lambda made log level an API type concern.

File: PowerShell/README.md (hunk 2 above)
Comment (ashishdhingra) [reply PowerShell/README.md]: Fixed. Please review.

**Review by normj — APPROVED** [review]

**Review by philasmar — APPROVED** [review]

**Pull request merged**